[2024-12-18T20:23:34.732+0000] {__init__.py:54} DEBUG - Loading core task runner: StandardTaskRunner
[2024-12-18T20:23:34.780+0000] {base_task_runner.py:73} DEBUG - Planning to run as the  user
[2024-12-18T20:23:34.792+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-12-18T20:23:34.808+0000] {taskinstance.py:2625} DEBUG - <TaskInstance: tiktok_dag.load_usernames manual__2024-12-18T20:23:32.949324+00:00 [queued]> dependency 'Previous Dagrun State' PASSED: True, The task did not have depends_on_past set.
[2024-12-18T20:23:34.809+0000] {taskinstance.py:2625} DEBUG - <TaskInstance: tiktok_dag.load_usernames manual__2024-12-18T20:23:32.949324+00:00 [queued]> dependency 'Task Instance Not Running' PASSED: True, Task is not in running state.
[2024-12-18T20:23:34.809+0000] {taskinstance.py:2625} DEBUG - <TaskInstance: tiktok_dag.load_usernames manual__2024-12-18T20:23:32.949324+00:00 [queued]> dependency 'Task Instance State' PASSED: True, Task state queued was valid.
[2024-12-18T20:23:34.810+0000] {taskinstance.py:2625} DEBUG - <TaskInstance: tiktok_dag.load_usernames manual__2024-12-18T20:23:32.949324+00:00 [queued]> dependency 'Trigger Rule' PASSED: True, The task instance did not have any upstream tasks.
[2024-12-18T20:23:34.817+0000] {taskinstance.py:2625} DEBUG - <TaskInstance: tiktok_dag.load_usernames manual__2024-12-18T20:23:32.949324+00:00 [queued]> dependency 'Not In Retry Period' PASSED: True, The task instance was not marked for retrying.
[2024-12-18T20:23:34.817+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: tiktok_dag.load_usernames manual__2024-12-18T20:23:32.949324+00:00 [queued]>
[2024-12-18T20:23:34.825+0000] {taskinstance.py:2625} DEBUG - <TaskInstance: tiktok_dag.load_usernames manual__2024-12-18T20:23:32.949324+00:00 [queued]> dependency 'Pool Slots Available' PASSED: True, There are enough open slots in default_pool to execute the task
[2024-12-18T20:23:34.826+0000] {taskinstance.py:2625} DEBUG - <TaskInstance: tiktok_dag.load_usernames manual__2024-12-18T20:23:32.949324+00:00 [queued]> dependency 'Previous Dagrun State' PASSED: True, The task did not have depends_on_past set.
[2024-12-18T20:23:34.829+0000] {taskinstance.py:2625} DEBUG - <TaskInstance: tiktok_dag.load_usernames manual__2024-12-18T20:23:32.949324+00:00 [queued]> dependency 'Trigger Rule' PASSED: True, The task instance did not have any upstream tasks.
[2024-12-18T20:23:34.832+0000] {taskinstance.py:2625} DEBUG - <TaskInstance: tiktok_dag.load_usernames manual__2024-12-18T20:23:32.949324+00:00 [queued]> dependency 'Task Concurrency' PASSED: True, Task concurrency is not set.
[2024-12-18T20:23:34.833+0000] {taskinstance.py:2625} DEBUG - <TaskInstance: tiktok_dag.load_usernames manual__2024-12-18T20:23:32.949324+00:00 [queued]> dependency 'Not In Retry Period' PASSED: True, The task instance was not marked for retrying.
[2024-12-18T20:23:34.833+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: tiktok_dag.load_usernames manual__2024-12-18T20:23:32.949324+00:00 [queued]>
[2024-12-18T20:23:34.834+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2024-12-18T20:23:34.855+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): load_usernames> on 2024-12-18 20:23:32.949324+00:00
[2024-12-18T20:23:34.866+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=3555) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-12-18T20:23:34.868+0000] {standard_task_runner.py:72} INFO - Started process 3560 to run task
[2024-12-18T20:23:34.867+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'tiktok_dag', 'load_usernames', 'manual__2024-12-18T20:23:32.949324+00:00', '--job-id', '88', '--raw', '--subdir', 'DAGS_FOLDER/tiktok_dag.py', '--cfg-path', '/tmp/tmp8mgg6lo1']
[2024-12-18T20:23:34.870+0000] {standard_task_runner.py:105} INFO - Job 88: Subtask load_usernames
[2024-12-18T20:23:34.872+0000] {cli_action_loggers.py:79} DEBUG - Calling callbacks: [<function default_action_log at 0x7f2a69185da0>]
[2024-12-18T20:23:34.956+0000] {task_command.py:467} INFO - Running <TaskInstance: tiktok_dag.load_usernames manual__2024-12-18T20:23:32.949324+00:00 [running]> on host 00d1ef865b53
[2024-12-18T20:23:35.016+0000] {taskinstance.py:2384} DEBUG - Clearing XCom data
[2024-12-18T20:23:35.034+0000] {retries.py:95} DEBUG - Running RenderedTaskInstanceFields.write with retries. Try 1 of 3
[2024-12-18T20:23:35.062+0000] {retries.py:95} DEBUG - Running RenderedTaskInstanceFields._do_delete_old_records with retries. Try 1 of 3
[2024-12-18T20:23:35.075+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='tiktok_dag' AIRFLOW_CTX_TASK_ID='load_usernames' AIRFLOW_CTX_EXECUTION_DATE='2024-12-18T20:23:32.949324+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-12-18T20:23:32.949324+00:00'
[2024-12-18T20:23:35.076+0000] {__init__.py:115} DEBUG - Preparing lineage inlets and outlets
[2024-12-18T20:23:35.077+0000] {__init__.py:154} DEBUG - inlets: [], outlets: []
[2024-12-18T20:23:35.078+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-12-18T20:23:35.101+0000] {tiktok_dag.py:43} INFO - Successfully loaded 3 usernames from /opt/***/dags/influencers.csv
[2024-12-18T20:23:35.102+0000] {tiktok_dag.py:44} INFO - Usernames loaded: ['mrbeast', 'jadeanna', 'feyenoord']
[2024-12-18T20:23:35.126+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-12-18T20:23:35.127+0000] {__init__.py:77} DEBUG - Lineage called with inlets: [], outlets: []
[2024-12-18T20:23:35.144+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-12-18T20:23:35.145+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=tiktok_dag, task_id=load_usernames, run_id=manual__2024-12-18T20:23:32.949324+00:00, execution_date=20241218T202332, start_date=20241218T202334, end_date=20241218T202335
[2024-12-18T20:23:35.162+0000] {cli_action_loggers.py:98} DEBUG - Calling callbacks: []
[2024-12-18T20:23:35.208+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-12-18T20:23:35.230+0000] {dagrun.py:968} DEBUG - number of tis tasks for <DagRun tiktok_dag @ 2024-12-18 20:23:32.949324+00:00: manual__2024-12-18T20:23:32.949324+00:00, state:running, queued_at: 2024-12-18 20:23:33.026697+00:00. externally triggered: True>: 2 task(s)
[2024-12-18T20:23:35.231+0000] {dagrun.py:989} DEBUG - number of scheduleable tasks for <DagRun tiktok_dag @ 2024-12-18 20:23:32.949324+00:00: manual__2024-12-18T20:23:32.949324+00:00, state:running, queued_at: 2024-12-18 20:23:33.026697+00:00. externally triggered: True>: 1 task(s)
[2024-12-18T20:23:35.232+0000] {taskinstance.py:2625} DEBUG - <TaskInstance: tiktok_dag.fetch_all_user_data manual__2024-12-18T20:23:32.949324+00:00 [None]> dependency 'Not In Retry Period' PASSED: True, The task instance was not marked for retrying.
[2024-12-18T20:23:35.232+0000] {taskinstance.py:2625} DEBUG - <TaskInstance: tiktok_dag.fetch_all_user_data manual__2024-12-18T20:23:32.949324+00:00 [None]> dependency 'Previous Dagrun State' PASSED: True, The task did not have depends_on_past set.
[2024-12-18T20:23:35.233+0000] {taskinstance.py:2613} DEBUG - Dependencies all met for dep_context=None ti=<TaskInstance: tiktok_dag.fetch_all_user_data manual__2024-12-18T20:23:32.949324+00:00 [None]>
[2024-12-18T20:23:35.242+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-12-18T20:23:35.246+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
